### ISSUE TITLE
Training Deep RankNet

### ISSUE THREAD

**Issue by iqbalnaved** [gyglim/video2gif_code#2]
Hi Michael,

Thanks so much for sharing your pretrained model and demo codes. I was wondering if it would be possible for you to share the network with the Huber loss function . I would like to train it with my own data for my research. Looking forward to your reply.

Thanks in advance.

Naved

**Comment by gyglim** [comment]
Hi

I did this during an internship and cannot access the training code now. But I have code for a standard rank loss, which should be easy to adapt:

https://gist.github.com/gyglim/bae2688b29b9e64ba333fa157fc2e54b
